microsite/src/main/tut/interpreters/index.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: docs
+title: "Interpreters"
+section: interpreters
+position: 2
+---
+
+# {{page.title}}
+
+Given a schema for a type `A`, we can derive operations on `A` from it. Intuitively, these operations are functions that take an `A` as argument, or return an `A`, or both. But this can be generalised as "some functor `F` acting on `A`". 
+
+In other words, interpreting schemas into a given functor `F` is coming up with a function from `Schema[X]` to `F[X]` for any type `X`. Such functions are called *natural transformations* and denoted `Schema ~> F` using scalaz.
+
+## HAlgebras
+
+Thanks to the recursive nature of the schema GADT, these interpreters are produced by *folding* a schema using an *algebra* (to use the terminology of recursion schemes). In general, an algebra for a functor `M` and *carrier* `A` is just a function `M[A] => A`. 
+
+But here things are slightly more complicated since `Schema` isn't a mere functor, but an *higher-order* functor. That is, a functor acting on functors.
+
+```scala
+sealed trait Schema[F[_], A]
```

Comment:
I’m curious why we need this `F[_]` type parameter.

Reply:
Yes that's exactly it. 

In "normal" recursion schemes, you would have something like `Schema[A]` where you would use `A` as your "recursive slot" (the children of your nodes).

But here we need to know a little bit more about our children (we need to know they are schema-y things that represent some type `A`).

So when we define things like `SumTerm[F[_], A](id: SumTermId, base: F[A]) extends Schema[F, A]`, the `F` can be anything, but it must "be about the (data) type `A`".

Does that make more sense?